code/game/gamemodes/miniantags/revenant/revenant_abilities.dm

```diff
@@ -199,20 +200,20 @@
 	var/shock_damage = 20
 	action_icon_state = "overload_lights"
 
-/obj/effect/proc_holder/spell/aoe_turf/revenant/overload/cast(list/targets, mob/living/simple_animal/revenant/user = usr)
+/obj/effect/proc_holder/spell/aoe_turf/revenant/overload/cast(list/targets, mob/living/simple_animal/revenant/user = usr) // todo: make this not as gross
 	if(attempt_cast(user))
 		for(var/turf/T in targets)
 			spawn(0)
 				for(var/obj/machinery/light/L in T.contents)
 					spawn(0)
```

Comment:
😬

Reply:
this scares me. I will try and make it shitn't no promises tho. I am not even sure how to swing at this? putting it inside of TWO `INVOKE_ASYNC`s sound gross